### ISSUE TITLE
How to open more than one market or pair?

### ISSUE THREAD

**Issue by jomagalo** [TDex-network/tdex-daemon#745]
I also don't see a way to open a certain market. If I create multiple markets and use `tdex market open` it only opens the first market created, but I don't see a way to call a market or pair by name in order to open it.

I suppose there should be some command like:

`tdex market open lbtc_usdt`
`tdex market open lbtc_eur`
or
`tdex market open --name lbtc_usdt`
`tdex market open --name lbtc_eur`

But it has no effect, it only opens the first market or pair created. 

In summary, with what commands can you refer to one market and another?

```
tdex market open -help
NAME:
   tdex operator CLI market open - open a market

USAGE:
   tdex operator CLI market open [command options] [arguments...]

OPTIONS:
   --help, -h  show help (default: false)
```

**Comment by tiero** [comment]
To switch market to operate on you use:

```
tdex config set base_asset 144c654344aa716d6f3abcc1ca90e5641e4e2a7f633bc09fe3baf64585819a49

tdex config set quote_asset f3d1ec678811398cd2ae277cbe3849c6f6dbd72c74bc542f7c4b11ff0e820958
```

then all following commands will refer to that market

https://dev.tdex.network/docs/latest/provider/daemon/market_account#create-a-market
